### ISSUE TITLE
Google Analytics functionality support

### ISSUE THREAD

**Issue by ramunl** [microg/GmsCore#2074]
Hey all,

May be somebody can give me  a hint hoow to support  Firebase Analytics events  working  with MicroG apk installed. 
After I researched the code a bit I've found that MicroG has stubs for Analytics just to avoid it's crashing and really MicroG doesn't intend to fully support it,  probably there is a way to bypass those stubs to call the FireBase native code. 
Any help would be very appreciated.

**Comment by sconim** [comment]
Yes, there is a way! Install Google Play Services and forget about microG. 

There is no such thing as convenience AND privacy. One must give way to the other. If this comes as a surprise to you, microG caters to privacy.

**Comment by VxlerieUwU** [comment]
Analytics are really intrusive and borderline spyware so no, it's a good thing that microg dosent support them
